### PR TITLE
Changes in Classes solved 

### PR DESCRIPTION
**Pull request by arev-dev**

### CODE REVIEW

File: YouCode.BE/Config.cs
Comment (eliseo-arevalo) [inline YouCode.BE/Config.cs:1]: eliminao

File: YouCode.GUI/Views/Shared/_Layout.cshtml
Comment (eliseo-arevalo) [inline YouCode.GUI/Views/Shared/_Layout.cshtml:1]: Que cambiaste aqui :o?

**Review by eliseo-arevalo — APPROVED** [review]
Todo bien pa

File: YouCode.BE/Config.cs
Comment (arev-dev) [reply YouCode.BE/Config.cs:1]: NoOoO

**Pull request merged**